src/app.module.ts

```diff
@@ -0,0 +1,23 @@
+import { BrowserModule } from "@angular/platform-browser";
+import { NgModule } from "@angular/core";
+
+import { CoreModule } from "@core"; // Import libraries
+import { AppRoutingModule } from "@router";
+import { AppComponent } from "@components/app";
+
+import { GaugeModule } from "angular-gauge";
+
+@NgModule({
+  declarations: [
+    AppComponent
+  ],
+  imports: [
+    BrowserModule.withServerTransition({ appId: "serverApp" }),
+    GaugeModule.forRoot(),
+    CoreModule,
+    AppRoutingModule,
+  ],
+  providers: [],
+  bootstrap: [AppComponent]
+})
+export class AppModule { }
```

Comment:
Do you consolidate your imports to just a few classes and then export it for use in other classes for code-cleanliness? I realize that was possible! Very cool.

Reply:
I don't remember tbh... iirc, this was a requirement of the Angular library.